packages/genui/a2ui-playground/lynx-src/a2ui/index.css

```
@@ -1,4 +1,81 @@
+  --a2ui-color-background-ambient: var(--canvas-ambient);
+  --a2ui-color-surface-muted: var(--neutral-film);
+  --a2ui-color-surface-strong: var(--paper-clear);
+  --a2ui-color-border-strong: var(--rule);
+  --a2ui-color-text: var(--content);
+  --a2ui-color-text-muted: var(--content-muted);
+  --a2ui-color-text-subtle: var(--content-muted-2);
+  --a2ui-color-primary-content-muted: var(--primary-content-faded);
+  --a2ui-color-overlay: var(--neutral-film);
+}
+
 page {
```

Stylelint (17.11.0)

[error] Unknown type selector "page" (selector-type-no-unknown)

(selector-type-no-unknown)